conan/tools/meson/helpers.py

```diff
@@ -0,0 +1,94 @@
+__all__ = ["to_meson_machine", "to_meson_value", "to_meson_cppstd"]
+
+# https://mesonbuild.com/Reference-tables.html#operating-system-names
+_meson_system_map = {
+    'Android': 'android',
+    'Macos': 'darwin',
+    'iOS': 'darwin',
+    'watchOS': 'darwin',
+    'tvOS': 'darwin',
+    'FreeBSD': 'freebsd',
+    'Emscripten': 'emscripten',
+    'Linux': 'linux',
+    'SunOS': 'sunos',
+    'Windows': 'windows',
+    'WindowsCE': 'windows',
+    'WindowsStore': 'windows'
+}
+
+# https://mesonbuild.com/Reference-tables.html#cpu-families
+_meson_cpu_family_map = {
+    'armv4': ('arm', 'armv4', 'little'),
+    'armv4i': ('arm', 'armv4i', 'little'),
+    'armv5el': ('arm', 'armv5el', 'little'),
+    'armv5hf': ('arm', 'armv5hf', 'little'),
+    'armv6': ('arm', 'armv6', 'little'),
+    'armv7': ('arm', 'armv7', 'little'),
+    'armv7hf': ('arm', 'armv7hf', 'little'),
+    'armv7s': ('arm', 'armv7s', 'little'),
+    'armv7k': ('arm', 'armv7k', 'little'),
+    'armv8': ('aarch64', 'armv8', 'little'),
+    'armv8_32': ('aarch64', 'armv8_32', 'little'),
+    'armv8.3': ('aarch64', 'armv8.3', 'little'),
+    'avr': ('avr', 'avr', 'little'),
+    'mips': ('mips', 'mips', 'big'),
+    'mips64': ('mips64', 'mips64', 'big'),
+    'ppc32be': ('ppc', 'ppc', 'big'),
+    'ppc32': ('ppc', 'ppc', 'little'),
+    'ppc64le': ('ppc64', 'ppc64', 'little'),
+    'ppc64': ('ppc64', 'ppc64', 'big'),
+    's390': ('s390', 's390', 'big'),
+    's390x': ('s390x', 's390x', 'big'),
+    'sh4le': ('sh4', 'sh4', 'little'),
+    'sparc': ('sparc', 'sparc', 'big'),
+    'sparcv9': ('sparc64', 'sparc64', 'big'),
+    'wasm': ('wasm32', 'wasm32', 'little'),
+    'x86': ('x86', 'x86', 'little'),
+    'x86_64': ('x86_64', 'x86_64', 'little')
+}
+
+_meson_vs_cppstd_map = {
+    '14': "'vc++14'",
```

Comment:
I am realizing this is already broken, containing the quoted strings. It should be a plain string.